src/main/java/frc/robot/Constants.java

```diff
@@ -1,23 +1,19 @@
 package frc.robot;
 
-public interface Constants {
-	/*
-	 * ALL MEMBERS ARE REQUIRED TO MAINTAIN THIS CLASS
-	 * 
-	 * This class is meant to store all non-electrical/port values so that they can easily be
-	 * modified and viewed here. (ints, doubles, booleans ONLY)
-	 * 
-	 * example:
-	 * 
-	 * public static final int VAL_OF_NUM = 0;
-	 * 
-	 * where
-	 * 
-	 * public = Accessible to all classes
-	 * static = Non-abstract reference
-	 * final = value can not be modified other than where it is declared
-	 */
-	// Controllers
-	public static final int PILOT_JOYSTICK = 0;
-	public static final int COPILOT_JOYSTICK = 1;
-}
\ No newline at end of file
+/*
+* ALL MEMBERS ARE REQUIRED TO MAINTAIN THIS CLASS This class is meant to
+* store all non-electrical/port values so that they can easily be modified
+* and viewed here. (ints, doubles, booleans ONLY) example: public static
+* final int VAL_OF_NUM = 0; where public = Accessible to all classes static
+* = Non-abstract reference final = value can not be modified other than
+* where it is declared
+*/
+public final class Constants {
+    // Controllers
+    public static final int PILOT_JOYSTICK   = 0;
+    public static final int COPILOT_JOYSTICK = 1;
+
+    public static final double FULL_PRESS_MARGIN = 0.99D;
```

Comment:
Seems like these 3 constants should live in OperatorInterface.java, since that is where they are used.

Reply:
I don’t know who put them there, I will move them to the OI